sdk/metricsadvisor/ai-metrics-advisor/src/transforms.ts

```diff
@@ -201,7 +201,7 @@ export function fromServiceMetricFeedbackUnion(
         feedbackType: "Anomaly",
         startTime: orig.startTime,
         endTime: orig.endTime,
-        value: orig.value.anomalyValue,
+        value: orig.value!.anomalyValue,
```

Comment:
can you double check whether all these  `!` assertions are needed?  I remember required-ness of some properties are fixed in the swagger.

Reply:
Yes they did it before but fxed it later. Thanks for pointing it, i removed the optional